### ISSUE TITLE
Judgement point is above the judgement area on non-default skins

### ISSUE THREAD

**Issue by peppy** [ppy/osu#8534]
![image](https://user-images.githubusercontent.com/191335/78044011-0e068200-73af-11ea-9239-5e5d72f21bf6.gif)

Judgement seems to be occurring above, rather than over the judgement area when not using the default skin. As seem on osu!classic above.

**Comment by mcendu** [comment]
Seems to be intended (Could be verified by setting the speed to 1 in stable).

**Comment by peppy** [comment]
The intention is that you should hit when it is 100% overlapping the judgement area. looks like stable implementation is also flawed..

**Comment by peppy** [comment]
![image](https://user-images.githubusercontent.com/191335/78089097-42a32980-7401-11ea-8378-4b3610f43e1b.png)

This makes it pretty obvious how incorrect stable is currently. For people using their own skins: how have you fixed this? Just be re-aligning your skin to match? It may be important to know this when implementing a fix.

**Comment by doughn0** [comment]
mania-stage-hint.png stays behind the notes and mania-key files should be in front of them. [YUGEN](https://osu.ppy.sh/community/forums/topics/365036) has the hit detection aligned the way you want it, but the notes go behind the judgement line (the bright judgement line is part of the mania-key images in this skin). Currently the judgement position in skin.ini is for the bottom of the note, so in yugen the images probably go over the real judgement line.

**Comment by peppy** [comment]
Yep, YUGEN also looks correct on lazer, so I believe this will need to be fixed specifically in the classic skin implementation.
